pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -49,42 +51,68 @@ func (h Handler) reconcilePCIDevices(nodename string) error {
 		return err
 	}
 
+	commonLabels := map[string]string{"nodename": nodename} // label
 	var setOfRealPCIAddrs map[string]bool = make(map[string]bool)
 	for _, dev := range pci.Devices {
 		setOfRealPCIAddrs[dev.Address] = true
 		name := v1beta1.PCIDeviceNameForHostname(dev, nodename)
 		// Check if device is stored
-		_, err := h.client.Get(name, metav1.GetOptions{})
+		devCR, err := h.client.Get(name, metav1.GetOptions{})
 
 		// TODO use k8s.io/apimachinery/apierrors IsNotFound
 		if err != nil {
-			logrus.Errorf("Failed to get %s: %s\n", name, err)
-
-			// Create the PCIDevice CR if it doesn't exist
-			var pdToCreate v1beta1.PCIDevice = v1beta1.NewPCIDeviceForHostname(dev, nodename)
-			logrus.Infof("Creating PCI Device: %s\n", err)
-			pdToCreate.Labels = map[string]string{"nodename": nodename} // label
-			_, err := h.client.Create(&pdToCreate)
-			if err != nil {
-				logrus.Errorf("Failed to create PCI Device: %s\n", err)
+			if apierrors.IsNotFound(err) {
+				logrus.Infof("[PCIDeviceController] Device does %s does not exist: %s\n", name)
```

Comment:
```suggestion
				logrus.Infof("[PCIDeviceController] Device %s does not exist: %s", name)
```

We don't need `\n` in the logging messages.

Reply:
Still have some redundant `\n` need to remove.

https://github.com/harvester/pcidevices/pull/20/files#diff-8c6dfb3b3b6b632d555b11e4d7d51828f55bfacd790ef5a42edd955a8437f692R73
https://github.com/harvester/pcidevices/pull/20/files#diff-8c6dfb3b3b6b632d555b11e4d7d51828f55bfacd790ef5a42edd955a8437f692R89